k-distribution/k-tutorial/1_basic/19_debugging/README.md

```diff
@@ -67,7 +73,12 @@ To make full advantage of the GDB features of K, you should follow the first
 command listed in this output message and add the corresponding
 `add-auto-load-safe-path` command to your `~/.gdbinit` file as prompted.
 Please note that the path will be different on your machine than the one
-listed above. 
+listed above. Adding directories to the "load safe path" effectively tells GDB
+to trust those directories. All content under a given directory will be recursively
+trusted, so if you want to avoid having to add paths to the "load safe path" every
+time you kompile a different `K` definition, then you can just trust a top-level
+directory containing all your kompiled files. More info on the load safe path
```

Comment:
I would not really recommend people do this because in most people's home directory, such a directory is likely to also contain code that you probably shouldn't just blindly trust.

Reply:
@dwightguth Happy to remove this portion from the PR, but in that case I think this statement from the `K` user manual should be re-thought:

>A good way to do this would be to pick a minimum top-level path that covers all of your kompiled semantics (ie. set auto-load safe-path ~/k-semantics)

(source: https://kframework.org/USER_MANUAL/).

Reply:
@dwightguth I'd like to get this PR closed off my TODO list. I've tweaked the wording in response to your comment.